src/test/java/org/assertj/core/util/StackTraceUtils.java

```diff
@@ -28,4 +28,10 @@ public static boolean hasStackTraceElementRelatedToAssertJ(Throwable throwable)
     return false;
   }
 
+  /**
+   * Returns regex which match stack trace.
+   */
+  public static String getStackTraceRegex() {
+    return "\\[(?s).*[\\n\\r].*\\]";
```

Comment:
let's avoid regex as much as we can, I only use regex when everything else fails since they are really unreadable and easy to screw up.